packages/file_selector/file_selector_macos/test/file_selector_macos_test.mocks.dart

```diff
@@ -0,0 +1,51 @@
+// Mocks generated by Mockito 5.3.2 from annotations
+// in file_selector_macos/example/macos/Flutter/ephemeral/.symlinks/plugins/file_selector_macos/test/file_selector_macos_test.dart.
+// Do not manually edit this file.
+
+// ignore_for_file: no_leading_underscores_for_library_prefixes
+import 'dart:async' as _i3;
+
+import 'package:file_selector_macos/src/messages.g.dart' as _i4;
+import 'package:mockito/mockito.dart' as _i1;
+
+import 'messages_test.g.dart' as _i2;
+
+// ignore_for_file: type=lint
```

Comment:
Is this method preferred to single line ignores?

Reply:
File-level ignores are pretty common in generated code. This is Mockito-generated code though so we don't control how ignores are handled (unless we hand-edit the generated code, which we avoid when possible).

Reply:
Ah, didn't realize this was generated code.